### PR TITLE
update

### PR DESCRIPTION
**Pull request by QuangDoo**
## Summary by CodeRabbit

- **New Features**
	- Logout functionality has been temporarily disabled.

- **Bug Fixes**
	- Corrected a typographical error in the token verification process to improve reliability.

### CODE REVIEW

File: src/routes/users.routes.ts [hunk 1]
```diff
@@ -43,7 +37,7 @@ usersRouter.post('/login', checkLoginUserExists, loginValidator, loginController
  * @author QuangDoo
  */
 
-usersRouter.post('/logout', accessTokenValidator, wrapRequestHanlder(logoutController))
+// usersRouter.post('/logout', accessTokenValidator, wrapRequestHanlder(logoutController))
```
[the remark this answers is not in the document]
Comment (QuangDoo) [reply src/routes/users.routes.ts:40]:
```suggestion
// usersRouter.post('/logout', accessTokenValidator, wrapRequestHanlder(logoutController))
```

File: src/utils/jwt.ts [hunk 2]
```diff
@@ -21,12 +21,12 @@ export const signToken = ({
...
-  return new Promise((resovle, reject) => {
+  return new Promise((resolve, reject) => {
     return jwt.verify(token, process.env.JWT_SECRET as string, (err, decoded) => {
       if (err) {
         throw reject(err)
       } else {
-        return resovle(decoded)
+        return resolve(decoded)
       }
     })
```
[the remark this answers is not in the document]
Comment (VanQuang03) [reply src/utils/jwt.ts]: @QuangDoo

File: src/routes/users.routes.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (VanQuang03) [reply src/routes/users.routes.ts:40]: @QuangDoo

**Review by VanQuang03 — CHANGES_REQUESTED** [review]

**Review by VanQuang03 — APPROVED** [review]
approved

**Pull request merged**